redhat-developer/developers.redhat.com
# [DEVELOPER-5589] Display proper moderation state by permission

Now, users with the 'view all revisions' permission will see the latest
revision state of the Product node when they visit the new Download and
Getting Started product pages, but anonymous users will continue to see
the default revision state (i.e. the last published state). This was
always the intended behavior of these product pages.

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-5589

### Verification Process

* Visit a product page and create a new Draft state and create some content in the new Product page assembly fields for both Content, Downloads and Getting Started.
* Check the 'Use New Product Pages' checkbox field.
* Submit the form.

* Visit the pages both logged in as a Content Creator and/or Admin (requires the 'view all revisions' permission) AND open an incognito window and visit the pages as an anonymous user
* You should see that you will view the latest revision of the content as the logged in user (Content Creator and/or Admin), but you will see the last published state of the Product as the anonymous user
